options/linux/include/sys/epoll.h

```diff
@@ -3,7 +3,7 @@
 
 // TODO: We only need sigset_t and not the whole signal.h
 #include <stdint.h>
-#include <signal.h>
+#include <abi-bits/signal.h>
```

Comment:
Why is this change needed? If this fixes the above TODO, could you remove it please?

Reply:
This both fixes the TODO and resolves problems with weston, where includes of this do seem correct.